.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,21 +35,18 @@ What do you think, how the system should have worked?
 
 ### Actual outcome
 
-What did acutally happen?
+What did actually happen?
 
 ### Configuration
 
-**privacyIDEA Version**:
-
-**Installation method**: (from source, github, virtualenv, ubuntu packages...)
-
-**Operating System**:
-
-**Webserver**:
-
-**Tokendatabase**: (like MySQL, PostgreSQL, ...)
+* **privacyIDEA version**:
+* **Installation method**: (from PyPI, Ubuntu packages, ...)
+* **Python version**:
+* **Operating system**:
+* **Webserver**:
+* **Token database**: (MySQL, PostgreSQL, ...)
 
 ### Log file
 
-**Turn on debug level and take a look at the privacyidea.log!**
-**If appropriate, attach the log file.**
+**Set PI_LOGLEVEL = logging.DEBUG in pi.cfg and take a look at the privacyidea.log!**
```

Comment:
Can we use `Code`-format in these templates? Might make the text more readable.

Reply:
Hm, I'm not sure if this would help, because users only see the text box without any formatting anyway (plus the preview, of course). Or maybe I am misinterpreting your comment :)